src/shared/onefx-auth/onefx-auth.ts

```diff
@@ -54,7 +55,19 @@ export class OnefxAuth {
     const link = `${this.config.emailTokenLink}${token}`;
     logger.debug(`sending out password reset email ${link}`);
 
-    const emailContent = t("auth/forgot_password.email_content", { link });
+    // const emailContent = t("auth/forgot_password.email_content", { link });
+    const emailContent = template({
+      logoSrc: "https://www.guanxilab.com/favicon.png",
+      forgotPasswordText: "Forgot your password?",
+      forgotPasswordDes:
+        "It happens to the best of us. The good news is you can change it right.",
```

Comment:
t("auth/forgot_password.email_content")

update the message to

Hi, we’ve received a request to reset your password. If you didn’t make the request, just ignore this message. Otherwise, you can reset your password using the following link.